### DISCUSSION TITLE
Is there a way to make the languages card show all languages?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2289] [Q&A] [answered]
Is there a way to make the languages card show all languages?

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori Thanks for your question. This is one of our top feature requests (see https://github.com/anuraghazra/github-readme-stats/issues/1651). However, we did not yet decide if we would like to increase the maximum number of languages shown on the card.
